checkov/terraform/checks/graph_checks/aws/AWSSSMParametershouldbeEncrypted.yaml

```diff
@@ -0,0 +1,15 @@
+metadata:
+  name: "AWS SSM Parameter should be Encrypted"
+  guidelines: "AWS Systems Manager (SSM) parameters that store sensitive data, for example, passwords, database strings, and permit codes are encrypted so as to meet security and compliance prerequisites. An encrypted SSM parameter is any sensitive information that should be kept and referenced in a protected way."
+  category: ""
+  severity: "medium"
```

Comment:
@C-Nandhini The `metadata` block should have an ID, following the convention `CKV2_AWS_...`

Reply:
@nimrodkor - This policy is part of Prisma and as part of feature parity we need this policy in BC. Can you share me the process to get the ID?

Reply:
Search the codebase for CKW_AWS_ - take the next number - currently ~199

Reply:
This is a graph check, so the next ID should be `CKV2_AWS_34` @C-Nandhini

Reply:
@nimrodkor - ID is added, please check.